gqlalchemy/reference/transformations/export.md

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_label: export
+title: gqlalchemy.transformations.export
+---
+
+# Abstract
+
+GQLAlchemy currently supports exporting data from Memgraph to NetworkX, DGL and PyG graph format.  
+
+## Transporter
+
+Abstract transporter that provides interface for translating Memgraph data into graph objects like `NetworkX, DGL and PyG`.
+
+## GraphTransporter
+
+`GraphTransporter` is used for converting Memgraph graph to the specific graph type offered by some Python package (PyG, DGL, NX...). Each `GraphTransporter` has a reference to the specific translator object which is used for translating Memgraph data into correct graph type. This translator does all the dirty stuff so check out docs about `DGLTranslator, PyGTranslator and NxTranslator` to find out implementation details.
+
+### Methods
+
+#### `__init__`
+
+```python
+def __init__(self, 
+    graph_type: str, 
+    default_node_label="NODE",
+    default_edge_type="RELATIONSHIP",
+    host: str = mg_consts.MG_HOST,
+    port: int = mg_consts.MG_PORT,
+    username: str = mg_consts.MG_USERNAME,
+    password: str = mg_consts.MG_PASSWORD,
+    encrypted: bool = mg_consts.MG_ENCRYPTED,
+    client_name: str = mg_consts.MG_CLIENT_NAME,
+    lazy: bool = mg_consts.MG_LAZY,
+    ) -> None
+```
+
+Creates neccessary translator object based on a graph type provided.
+
+##### Input
+- `graph_type: str` -> Graph source type object. Can be any of the: [`DGL`, `NX`, `PYG`]
+- `default_node_label: str=NODE` -> Default node label that will be given to nodes when no other can be inferred.
+- `default_edge_type: str=RELATIONSHIP` -> Default edge type that will be given to edges when no other can be inferred.
+- `host: str=127.0.0.1` -> Host connection info for connecting to the Memgraph instance.
+- `port: int=7687` -> Port connection info for connecting to the Memgraph instance.
+- `username: str=""` -> Username connection info for connecting to the Memgraph instance.
+- `password: str=""` -> Password connection info for connecting to the Memgraph instance.
+- `encrypted: bool=False` -> Encrypted flag for connecting to the Memgraph instance.
+- `client_name: str=GQLAlchemy` -> Client name used for connecting to the Memgraph instance.
+- `lazy: bool=False` -> Lazy flag for connecting to the Memgraph instance.
```

Comment:
```suggestion
- `graph_type: str` -> Graph source type object. Can be one of the following: [`DGL`, `NX`, `PYG`]
- `default_node_label: str=NODE` -> Default node label that will be given to nodes when no other can be inferred.
- `default_edge_type: str=RELATIONSHIP` -> Default edge type that will be given to edges when no other can be inferred.
- `host: str=127.0.0.1` -> Host connection info for connecting to the Memgraph instance.
- `port: int=7687` -> Port connection info for connecting to the Memgraph instance.
- `username: str=""` -> Username connection info for connecting to the Memgraph instance.
- `password: str=""` -> Password connection info for connecting to the Memgraph instance.
- `encrypted: bool=False` -> Encrypted flag for connecting to the Memgraph instance.
- `client_name: str=GQLAlchemy` -> Client name used for connecting to the Memgraph instance.
- `lazy: bool=False` -> Lazy flag for connecting to the Memgraph instance.
```

Reply:
maybe it can be more than one??

Reply:
Not sure what you meant, what can be more than one?

Reply:
I rewrote it as "one of the following" but your construct of "any of" is making me think it I could use more parameters? but probably not